### ISSUE TITLE
Background not changing and Sticker not showing

### ISSUE THREAD

**Issue by ZaZa101101** [doki-theme/doki-theme-vscode#156]
I installed Doki theme on Linux Mint 20.3 Cinnamon and I selected a theme but for some reason the background image is not changing when I change between themes, even after reloading the screen and given the permissions that it request
https://youtu.be/aq-zZJuJN3w
Youtube link to a video with the issue because GitHub says it's too big to upload

**Comment by ZaZa101101** [comment]
Yes, it worked. Thanks a lot, I didn't understand why it didn't work, I had no issue before on Windows, but Linux seems to be a bit different I guess. Thanks again

**Comment by leoshusar** [comment]
I've got an idea, which may or may not work.
What about adding new command menu entry like "change Doki theme", there you would have all Doki themes and the selection would set the theme and also sticker & wallpaper? It would be alternative to the native VSC theme selector.

**Comment by Unthrottled** [comment]
> selection would set the theme

@leoshusar I like the idea, however the crux if the issue is that I _cannot_ set the theme at the moment. If I could then I would have done something like that :) 

If I get some time later, I might look into it again since they released this feature a while ago, there might be an API for setting the theme now. https://code.visualstudio.com/docs/getstarted/themes#_color-themes-from-the-marketplace

**Comment by leoshusar** [comment]
Ah, I see.
I don't have any experience with VSC API, but I found [this](https://stackoverflow.com/a/65297166/6641322), maybe you've seen it, but it looks like it might work.

**Comment by Unthrottled** [comment]
That was going to be one of the options I would pursue (updating the settings), feels hacky, but it might work.
